app/src/main/res/layout/fragment_example.xml

```diff
@@ -10,86 +10,83 @@
 		android:orientation="vertical"
 		android:padding="@dimen/spacing_large">
 
-		<TextView
-			android:id="@+id/title"
+		<ImageView
+			android:id="@+id/logo"
 			android:layout_width="wrap_content"
 			android:layout_height="wrap_content"
-			android:text="@string/example_title"
-			android:textSize="@dimen/font_medium_extra"
-			app:layout_constraintBottom_toTopOf="@id/usernameInput"
+			android:src="@mipmap/wolox_logo_large"
+			app:layout_constraintBottom_toTopOf="@id/etUsername"
 			app:layout_constraintEnd_toEndOf="parent"
 			app:layout_constraintStart_toStartOf="parent"
 			app:layout_constraintTop_toTopOf="parent"
 			app:layout_constraintVertical_bias="0" />
 
 		<EditText
-			android:id="@+id/usernameInput"
+			android:id="@+id/etUsername"
```

Comment:
Since we are using Viewbinding, you don't need to use hungarian notation for the IDs. Just `userName` is ok 👍🏽